techmatters/hrm
# CHI-2699 Endpoint for use in generalized search for contacts

## Description
- Implemented a POST endpoint. 

### Checklist

### Other Related Issues

None

### Verification steps
- Change line 140 in contactRoutes with ids present in your local db. 
- run `npm run build` and start service. 
- run curl command with POST method and `/contacts/searchV2` 
- Check that you get a valid response with the correct count and contacts

### AFTER YOU MERGE

1. Cut a release tag using the GitHub workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P
